swagger.yaml

```diff
@@ -88,38 +88,78 @@ definitions:
         items:
           $ref: '#/definitions/Ereignis'
     title: Ereignisse
+  Partner:
+    type: "object"
+    properties:
+      partnerId:
+        type: "string"
+      nachname:
+        type: "string"
+      vorname:
+        type: "string"
+      email:
+        type: "string"
+      telefonnummer:
+        type: "string"
+      mobilTelefon:
+        type: "string"
+      fax:
+        type: "string"
+      firmierung:
+        type: "string"
+      _links:
+        $ref: "#/definitions/PartnerLinks"
+    title: "Partner"
   Ereignis:
     type: object
     properties:
       zeitpunkt:
         type: string
         format: date-time
         description: Zeitpunkt des Ereignisses
-      akteur:
-        $ref: '#/definitions/Link'
+      quelle:
+        $ref: '#/definitions/Partner'
       meldung:
         type: string
         description: Beschreibung des Ereignisses
       ereignistyp:
         type: string
         enum:
-        - DOKUMENT
-        - STATUS
+        - DOKUMENTE
+        - STATUS_AENDERUNG
         - WIEDERVORLAGE
         - KOMMENTAR
-        - BEARBEITERWECHSEL
         - KOMMUNIKATION
         - HINWEIS
         - PRIORITAET
         - EIGENE_VORGANGSNUMMER
+      dokumente:
+        type: array
+        items:
+          $ref: '#/definitions/DokumenteLinks'
     title: Ereignis
+  DokumenteLinks:
```

Comment:
DokumenteLinks -> DokumentenLinks

Reply:
Das funktioniert so mit den Links?

in der Dokumente API ist es so umgesetzt:
![image](https://user-images.githubusercontent.com/570380/45369163-906cfc80-b5e5-11e8-9341-edf3ddd422e2.png)